app/assets/stylesheets/user_cabinets.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the UserCabinets controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
remove if not needed

Reply:
there will be many styles for this page in the future, so we may keep this file for now